cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -296,29 +300,11 @@ function Create-Directories {
 # Downloads some external helper scripts needed by other functions in this
 # module.
 function Download-HelperScripts {
-  if (-not (ShouldWrite-File ${env:K8S_DIR}\hns.psm1)) {
-    return
-  }
-  MustDownload-File -OutFile ${env:K8S_DIR}\hns.psm1 `
-    -URLs "https://www.googleapis.com/storage/v1/b/gke-release/o/winnode%2fconfig%2fsdn%2fmaster%2fhns.psm1?alt=media"
-}
-
-# Takes the Windows version string from the cluster bash scripts (e.g.
-# 'win1809') and returns the correct label to use for containers on this
-# version of Windows. Returns $null if $WinVersion is unknown.
-function Get_ContainerVersionLabel {
-  param (
-    [parameter(Mandatory=$true)] [string]$WinVersion
-  )
-  # -match does regular expression matching.
-  if ($WinVersion -match '1809') {
-    return '1809'
-  }
-  elseif ($WinVersion -match '2019') {
-    return 'ltsc2019'
+  if (ShouldWrite-File ${env:K8S_DIR}\hns.psm1) {
+    MustDownload-File `
+        -OutFile ${env:K8S_DIR}\hns.psm1 `
+        -URLs "https://www.googleapis.com/storage/v1/b/gke-release/o/winnode%2fconfig%2fsdn%2fmaster%2fhns.psm1?alt=media"
```

Comment:
better to use https://storage.googleapis.com/gke-release/winnode/config/sdn/master/hns.psm1

Reply:
Done.